dvc/remote/base.py

```diff
@@ -142,6 +138,21 @@ def _check_requires(self, config):
         ).format(url, missing, " ".join(missing), self.scheme)
         raise RemoteMissingDepsError(msg)
 
+    def _get_checksum_jobs(self, repo, config):
+        checksum_jobs = config.get(Config.SECTION_REMOTE_CHECKSUM_JOBS)
+        if not checksum_jobs:
+            if repo:
```

Comment:
Since you are catching `AttributeError`, you could remove the `if/else`. If there's no `repo` it is going to do `None.config` and fallback to `self.CHECKSUM_JOBS`.